### DISCUSSION TITLE
vercel is not pulling the last commit

### DISCUSSION THREAD

**Discussion by gabrielmendesdev** [vercel/community#3324] [Bug Report]
### Page to Investigate

https://vercel.com/gabrielmendes06/portifolio

### Steps to Reproduce

When updating my code and applying the push to my gitHub repository, all changes are applied to my repository, however, vercel cannot pull this latest update, and replicates the old code in my link

**Comment by constantinmusca** [comment]
No deployment is triggered anymore for me as well for https://vercel.com/constantinmusca/scolarel when I push. No preview deployment is triggered when creating PR and no prod deployment is triggered when I push to master.

**Comment by amyegan** [comment]
Sorry you're having deployment trouble. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give me a better view of what is going wrong.

**Comment by m0x61h0x64i** [comment]
@amyegan, i got same problem when i have changed my vercel.json :

```
{
    "version": 2,
    "builds": [
        {
            "src": "./api/index.js",
            "use": "@vercel/node"
        }
    ],
    "routes": [
        {
            "src": "/(.*)",
            "dest": "/"
        }
    ],
    "functions": {
        "api/index.js": {
            "memory": 1024,
            "maxDuration": 10
        }
    }
}
```
